packages/mobile/scripts/sync_branding.sh

```diff
@@ -21,7 +21,7 @@ echo $mobile_root
 cd "$mobile_root"
 
 # Please update the sha when valora branding updates are needed
-valora_branding_sha=74d23db
+valora_branding_sha=c678332
```

Comment:
Why did Valora branding need to change for this PR?

Reply:
I removed `remoteNotification` (moved remotely) which was in the Valora branding.